Gopkg.toml

```diff
@@ -98,6 +102,34 @@ required = [
   branch = "master"
   name = "golang.org/x/oauth2"
 
+[[override]]
+  name = "github.com/giantswarm/k8sportforward"
+  branch = "kubernetes-1.15.4"
```

Comment:
This is to be updated to "master" once https://github.com/giantswarm/k8sportforward/pull/19 gets merged.